scipy/integrate/_ivp/ivp.py

```diff
@@ -426,6 +426,46 @@ def solve_ivp(fun, t_span, y0, method='RK45', t_eval=None, dense_output=False,
      [8.         7.5534414  4.25308709 1.73277247 0.72069014 0.29932181
       0.12428631 0.05403123]]
 
+    A couple examples of using solve_ivp to solve differential functions with
+    complex matrix A
```

Comment:
```suggestion
    complex matrix ``A``.
```

Reply:
Thanks Pamphile! And no worries with the set up! I have made the changes!